src/libraries/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Rss20FeedFormatter.cs

```diff
@@ -585,9 +589,22 @@ private void ReadPerson(XmlReader reader, SyndicationPerson person)
             reader.ReadStartElement();
             if (!isEmpty)
             {
-                string email = reader.ReadString();
+                string s = reader.ReadString();
                 reader.ReadEndElement();
-                person.Email = email;
+                
+                if (!string.IsNullOrWhiteSpace(s))
+                {
+                    var m = PersonDisplayRegex.Match(s);
+                    if (m.Success && m.Groups.Count == 3)
+                    {
+                        person.Email = m.Groups[1].Value;
+                        person.Name = m.Groups[2].Value;
+                    }
+                    else
+                    {
+                        person.Email = s;
+	    			}
+    			}
```

Comment:
Nit: formatting

Reply:
Ouch, thanks!
It should be fixed now, hopefully (I hope, I'm waiting for the checks to finish).